src/group_cmd.cpp

```diff
@@ -398,13 +399,13 @@ CommandCost CmdDeleteGroup(DoCommandFlag flags, GroupID group_id)
 		OrderBackup::ClearGroup(g->index);
 
 		/* If we set an autoreplace for the group we delete, remove it. */
```

Comment:
This comment no longer applies to the entire `if` block and should move down a couple lines, I think. 🙂